src/getCypressElementCoordinates.ts

```diff
@@ -63,6 +66,13 @@ export function getCypressElementCoordinates(
     );
   }
 
+  // @ts-expect-error Cypress.config('scrollBehavior') type not defined in Cypress <6.1.0
+  const effectiveScrollBehavior = scrollBehavior ?? Cypress.config('scrollBehavior') ?? "center";
```

Comment:
Cypress's default scrollBehavior is "top", and has been for a while- this library's default was "center". 

Changing this to fall back to the "scrollBehavior" from the Cypress config could be a breaking change in Cypress >= 6.1, which would switch the behavior from center to top. Before 6.1.0, that config should be `undefined`.